Jenkinsfile

```diff
@@ -24,6 +32,7 @@ pipeline
             steps
             {
                 sh """
+                    apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.4/main/ maven;
```

Comment:
@khalilj without this Jenkins cannot find "nvm". I also added this here: https://github.com/Blazemeter/jenkins-docker-agent/blob/WAP-15611-migrate-jenkins-GKE/Dockerfile
But it didn't help. Is there a cleaner way?